firmware/1.5.4.1/app/include/user_config.h

```diff
@@ -107,8 +107,8 @@ extern void luaL_assertfail(const char *file, int line, const char *message);
  * if WIFI_STA_HOSTNAME_APPEND_MAC defined: Hostname MUST be 26 chars or less, since last 3 octets of MAC address will be appended
  * if defined hostname is invalid: hostname will default to NODE-xxxxxx (xxxxxx being last 3 octets of MAC address)
 */
-#define WIFI_STA_HOSTNAME "AlarmPanel"
-//#define WIFI_STA_HOSTNAME_APPEND_MAC
+#define WIFI_STA_HOSTNAME "konnected-security-"
```

Comment:
Yes, will add the IP address of the wifi setup to the docs. This was one of the confusing parts with my neighbors setup.